lib/cluster.js

```diff
@@ -728,6 +728,14 @@ function sendHelper(proc, message, handle, cb) {
 // to the callback but intercepts and redirects ACK messages.
 function internal(worker, cb) {
   return function(message, handle) {
+    if (message.cmd === 'NODE_WARN') {
```

Comment:
Yes, which is the route I had originally taken. I added this so that there could be more consistency in how those are handled... but that may be a premature optimization and I'm fine with pulling this back out for now.